### PR TITLE
[editor] Migrate mapsme to omaps

### PR DESCRIPTION
**Pull request by Jean-BaptisteC**
This PR following works started here #4050

### CODE REVIEW

**Review by biodranik — CHANGES_REQUESTED** [review]
The code doesn't load an old config. To reproduce:
1. Build the version from master (desktop or mobile)
2. Run and make an edit (you don't need to upload it to OSM, so no logging in is necessary)
3. Close the app. Build this branch.
4. Run this branch version.
5. Observe the ASSERT crash:
```
LOG TID(1) ERROR     0.29318 editor/config_loader.cpp:154 LoadFromLocal(): Config can not be loaded.
```

Obviously, the loading code should check if either the old or new XML element is present in the root node. And always save a new value.

File: editor/osm_editor.cpp [hunk 1]
```diff
@@ -168,7 +170,11 @@ void Editor::LoadEdits()
   m_features.Set(make_shared<FeaturesContainer>());
   auto loadedFeatures = make_shared<FeaturesContainer>();
 
-  for (auto const & mwm : doc.child(kXmlRootNode).children(kXmlMwmNode))
+  auto rootNode = doc.child(kXmlNewRootNode);
+  // Migrate clients with an old root node.
+  if (!rootNode)
+    rootNode = doc.child(kXmlOldRootNode);
```
Comment (vng) [inline editor/osm_editor.cpp:176]: - I don't mind. Delete kXmlOldRootNode var and put _old_ string manually here?
- add CHECK(rootNode, ());

**Review by vng — APPROVED** [review]

**Comment by biodranik** [comment]
@Jean-BaptisteC please test quickly if it works.

**Comment by Jean-BaptisteC** [comment]
Build failed
```
home/runner/work/organicmaps/organicmaps/editor/osm_editor.cpp:209:36: error: use of undeclared identifier 'kXmlNewRootNode'; did you mean 'kXmlRootNode'?
    xml_node root = doc.append_child(kXmlNewRootNode);
                                     ^~~~~~~~~~~~~~~
                                     kXmlRootNode
  /home/runner/work/organicmaps/organicmaps/editor/osm_editor.cpp:44:24: note: 'kXmlRootNode' declared here
  constexpr char const * kXmlRootNode = "omaps";
```

**Comment by biodranik** [comment]
Uuups ) fixed.

**Comment by Jean-BaptisteC** [comment]
Pixel 6 - Android 13 ✅

I have create 2 APK
- One from master branch
- One from this branch

I have installed on my device, the first version, i have update POI without Internet
I have update app with version from this branch, i have enabled Internet -> Change was commit on OSM

I have add contact:facebook on this POI https://www.openstreetmap.org/node/251474108

**Review by biodranik — APPROVED** [review]
Thank you for the PR and for testing!

**Pull request merged**